keepassxc-browser/background/event.js

```diff
@@ -87,7 +87,7 @@ kpxcEvent.onShowNotification = function(callback, tab, message) {
 
 kpxcEvent.showStatus = function(configured, tab, callback) {
     let keyId = null;
-    if (configured) {
+    if (configured && keepass.databaseHash !== 'no-hash') {
```

Comment:
I do not like "magic values". Where is this set? Wouldn't a blank databaseHash be sufficient and just as telling?

Reply:
Probably. This is originally from chromeIPass. Basically `no-hash` means the database is locked.

Reply:
Yeah i kind of figured, we can look at it later if necessary

Reply:
I changed the "magic value" to an empty string.